### PR TITLE
Fix get current surface for Android if use SDL

### PR DESCRIPTION
**Pull request by HadesD**
On android, i'm using SDL for create window
But without this, app will be crash, because SDL already created surface, so just get the surface from current

### CODE REVIEW

File: RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp [hunk 1]
```diff
@@ -119,7 +119,9 @@ namespace Ogre {
                                 "currentGLContext was specified with no current GL context",
                                 "EGLWindow::create");
                 }
-
+#if defined(OGRE_BITES_HAVE_SDL) && OGRE_BITES_HAVE_SDL
```
Comment (paroj) [inline RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp:122]: do not use defines here, but rather pass `currentGLSurface=true` via miscParams. See the `currentGLContext` code just above.
You can use #ifdef in OgreBites to conditionally pass this flag.

Comment (HadesD) [reply RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp:122]: @paroj 
where should I put `if (currentGLSurface=true)`?
is inside the `if (currentGLContext = true)` above accepable?

File: RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp [hunk 2]
```diff
@@ -120,6 +120,11 @@ namespace Ogre {
                                 "EGLWindow::create");
                 }
 
+                if((opt = miscParams->find("currentEGLSurface")) != end)
```
Comment (paroj) [inline RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp:123]: also add the parseBool part, as currently `currentEGLSurface=false` would be accepted

Comment (HadesD) [reply RenderSystems/GLSupport/src/EGL/Android/OgreAndroidEGLWindow.cpp]: @paroj 
Thanks,
I've added

**Comment by paroj** [comment]
one final request:

please add the new property to the property table:
https://github.com/OGRECave/ogre/blob/185c369e8f2bcdd98ade5e7649c4d11bd6db258b/OgreMain/include/OgreRenderSystem.h#L395

**Pull request merged**